init.js

```diff
@@ -11,20 +11,20 @@
 // If you haven't already, invoke `cmd+h tutorial` to read the tutorial first!
 
 // The current implementation treats the user's init file as if it were semi-dynamically-scoped:
-// The `~/init.js` file may mention `E, commands, vscode` with no ceremonial import of any kind!
+// The `~/.init.js` file may mention `E, commands, vscode` with no ceremonial import of any kind!
 // (This is similar to the use of the keyword `this` in object-oriented programming:
 //  It's an implicitly introduced argument!)
 
 // Below are sample fragments to help you get started; all the best!
 // (More honestly, this is Musa's personal init.js being shared with the world.)
 
 // It is encouraged to keep this file under version control, then make a symbolic link;
-// e.g.,   ln  -s  ~/my-cool-repo/init.js  ~/init.js
+// e.g.,   ln  -s  ~/my-cool-repo/init.js  ~/.init.js
 
 // ==================================================================================
 // ========= 🚀 Whenever we open VSCode, let's see a motivating message! 💪 ==========
 // ==================================================================================
-let welcome = `Welcome ${process.env.USER}! Today is ${E.shell('date')}!`
+let welcome = `Welcome ${E.username()} on ${process.platform}! Today is ${E.date()}!`
```

Comment:
I'm not sure about this change.

----

Let me defend the original, and then you defend the alteration and we'll take it from there 😁

----

The original is short & sweet:
- It involves only 2 interpolations.
- The first demonstrates the easy access to `process` ---which would be nice for existing NodeJS users to see.
- The second demonstrates the easy access to running shell commands ---which is a powerful feature for people to see right away.

While the alteration does a good job of lifting there to functions, for increased readability; I worry that the abstractions belie/obscure the power of the `E` API itself.

Reply:
Because the environmental variables are different under individual platform, .e.g. it's USER under unix like system while it's USERNAME under windows.